frontend/app/src/pages/main/events/index.tsx

```diff
@@ -501,7 +501,7 @@ function EventsTable() {
 
 function ExpandedEventContent({ event }: { event: Event }) {
   return (
-    <DialogContent className="w-fit max-w-[80%] min-w-[500px]">
+    <DialogContent className="w-fit max-w-[700px] overflow-hidden">
```

Comment:
I figure it doesn't matter much, but do we think about responsiveness at all (e.g. mobile / tablet)?

Reply:
That's a good callout -- we have tried to make login + some of the basic views responsive, but it's not been a priority